### PR TITLE
implements Rosenbaum's Golem.

### LINKED ISSUE #112: TCaR - Rosenbaum's Golem

### PR DESCRIPTION
**Pull request by stopfstedt**
fixes #112

### CODE REVIEW

File: server/game/cards/11-TCaR/RosenbaumsGolem.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                cardType: ['dude'],
+                gameAction: 'boot'
+            },
+            message: context => this.game.addMessage('{0} uses {1} to boot {2}', context.player, this, context.target),
+            handler: context => {
+                this.game.resolveGameAction(GameActions.bootCard({ card: context.target }), context).thenExecute(() => {
+                    context.game.promptForYesNo(context.player, {
+                        title: 'Do you want to give ' + context.target.title + ' +5 value?',
+                        onYes: () => {
+                            this.game.addMessage('{0} uses {1} to give {2} +5 value', context.player, this, context.target);
```
Comment (mmeldo) [inline server/game/cards/11-TCaR/RosenbaumsGolem.js:22]: The +5 value effect is missing.

**Review by mmeldo — CHANGES_REQUESTED** [review]

File: server/game/cards/11-TCaR/RosenbaumsGolem.js (hunk 1 above)
Comment (stopfstedt) [reply server/game/cards/11-TCaR/RosenbaumsGolem.js:22]: oh, doh! i'll fix it over lunch today.

Comment (mmeldo) [reply server/game/cards/11-TCaR/RosenbaumsGolem.js:22]: 😄

Comment (stopfstedt) [reply server/game/cards/11-TCaR/RosenbaumsGolem.js:28]: added the missing effect :heavy_check_mark:

**Review by mmeldo — APPROVED** [review]

**Pull request merged**